### PR TITLE
Version number tags1

### PR DESCRIPTION
**Pull request by AdrianLxM**
branch and date

again:
experimental won't show if not in repository "xDrip-Experimental"
branch will not show if it is "master".

It would be nice if anyone who has a windows system could test it and comment or if he/she has merging privileges and it works, merge it.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -14,6 +14,59 @@ repositories {
...
+        if(!branch.equals("master")) {
+            stringBuilder.append('-')
+            stringBuilder.append(branch)
+        }
+
+    } catch (ignored) {
+        return "NoGitSystemAvailable";
+    }
+    stringBuilder.append('-')
+    stringBuilder.append((new Date()).format('yyyy/MM/dd/HH:mm:ss'))
```
Comment (StephenBlackWasAlreadyTaken) [inline app/build.gradle]: I worry that drilling down to the second could cause a ton of noise in crashlytics, I think going down to the day should be more than sufficient here!

**Comment by StephenBlackWasAlreadyTaken** [comment]
Awesome stuff, just that one comment and I would say this is mergable!

**Comment by AdrianLxM** [comment]
Shortened the date:
branch and date

Maybe the 6th of June hadn't been a good test date for the screenshots: Is it ok to have Year/Month/Day ?

**Comment by StephenBlackWasAlreadyTaken** [comment]
Yeah, If you use `.` instead of `/` it would look nice in the year/month/date format

**Comment by StephenBlackWasAlreadyTaken** [comment]
Great!! Thanks a ton for this!

**Pull request merged**